package/utils/busybox/Config-defaults.in

```diff
@@ -2124,6 +2124,7 @@ config BUSYBOX_DEFAULT_WATCHDOG
 	default n
 config BUSYBOX_DEFAULT_FEATURE_IPV6
 	bool
+	depends on IPV6
```

Comment:
Looks cleaner to make the default value of BUSYBOX_DEFAULT_FEATUTURE_IPV6 dependant on IPV6; see https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=ec9576607d19ddda04334142bdd7feef8cb5c5ca